test/inventory.js

```diff
@@ -1,3 +1,4 @@
+process.env.ARBORIST_DEBUG = '1'
```

Comment:
is this being added to a couple tests deliberate? should it be set in all of the test files, or is this something that we get some specific benefit out of in these ones?

Reply:
I turned off debugging in the buildIdealTree tests on Windows, and everywhere by default on GH Actions, because it's (as of this refactor) just abusively slow to snapshot the entire tree every time we call CanPlaceDep, to the point where tests fail because they time out.

Reply:
e9212774d377bbb3ee9c230a8d15b8460f49c128